editor.planx.uk/src/components/Header.tsx

```diff
@@ -383,15 +415,38 @@ const EditorToolbar: React.FC<{
   );
 };
 
+interface ToolbarProps {
+  headerRef: RefObject<HTMLDivElement>;
+  team?: Team;
+}
+
+const Toolbar: React.FC<ToolbarProps> = ({ headerRef, team }) => {
+  const route = useCurrentRoute();
+  const path = route.url.pathname.split("/").slice(-1)[0];
+  const flowSlug = useStore((state) => state.flowSlug);
+
+  switch (path) {
```

Comment:
This isn't a great solution - clearly it's not very scalable.

I think ideally when setting a view doing some route handling, we could set a context of some sort (e.g. "Published", "Standalone", "Editor") which the UI could then reference in various ways.

I did try using `previewEnvironment` for this exact purpose but it's being used pretty inconsistently and a wider refactor is outside the scope of this PR. For example, `previewEnvironment = "editor"` when in `/unpublished` routes, and is set to `"standalone"` when on the list of teams.